projects/components/date-time-input/src/date-time-input.module.ts

```diff
@@ -0,0 +1,16 @@
+import { CommonModule } from '@angular/common';
+import { NgModule } from '@angular/core';
+import { FormsModule } from '@angular/forms';
+import { MatDatepickerModule } from '@angular/material/datepicker';
+import { MatInputModule } from '@angular/material/input';
+
+import { PsDateTimeInputComponent } from './date-time-input.component';
+
+export const dateTimeModuleImports = [CommonModule, MatDatepickerModule, FormsModule, MatInputModule];
```

Comment:
Doesn't seem right that this is an export const. const would make sense if the module uses it multiple times but why is this exported? Please check this